plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcMetadataSessionProperties.java

```diff
@@ -46,6 +47,11 @@ public JdbcMetadataSessionProperties(JdbcMetadataConfig jdbcMetadataConfig, @Max
                         "Enable aggregation pushdown",
                         jdbcMetadataConfig.isAggregationPushdownEnabled(),
                         false))
+                .add(booleanProperty(
+                        CASE_INSENSTIVE_PREDICATE_PUSHDOWN_ENABLED,
+                        "Enable case insensitive predicate pushdown",
```

Comment:
I'm wondering if this is a good description for this option. As I understand it, pushdown is disabled on character columns (for all operators?), on databases which are doing case insensitive string comparison by default, right? Maybe something "Always do potentially unsafe (var)char predicate pushdown on databases which do case insensitive string comparison"?

Also, this is a tricky setting, or perhaps I should say "esoteric", so can it be documented somewhere is better detail?

Reply:
It's OK now, though one could argue that "potentially unsafe" already implies "may cause incorrect results"